compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -546,13 +551,14 @@ void buildAMDAIETransformPassPipeline(OpPassManager &variantPassManager) {
 // TODO (Erwei): The "packPeel" temporary argument should be removed once
 // pack-peel and pack-pad share the same pass pipeline. See TODOs inlined below
 // for details.
-void addMLIRAIRLoweringPasses(OpPassManager &passManager, bool packPeel) {
+void addMLIRAIRLoweringPasses(OpPassManager &passManager,
+                              AIEPassPipeline passPipeline) {
   // Add passes for preparing for lowering to MLIR-AIR
   passManager.addPass(createEraseHALDescriptorTypeFromMemRefPass());
   passManager.addPass(memref::createFoldMemRefAliasOpsPass());
   passManager.addPass(createAMDAIEBridgeToAIRPass());
   // TODO (Erwei): Figure out a way to work with AMDAIEPackToDmaPass.
-  if (packPeel)
```

Comment:
If this flag is not used anymore, we should also clean up the flag at the top of this file.

Reply:
Although this variable packPeel is no longer used, the AIR pipeline for pad-pack, pack-peel and conv-decompose still differ in some places, which were implemented with some if conditions.

Reply:
NVM... I thought there was a separate flag, but looks like there's not.